public/app/core/table_model.ts

```diff
@@ -41,7 +41,8 @@ export default class TableModel implements TableData {
   }
 
   sort(options: { col: number; desc: boolean }) {
-    if (options.col === null || this.columns.length <= options.col) {
+    // Since 8.3.0 col property can be also undefined
+    if (options.col === null || options.col === undefined || this.columns.length <= options.col) {
```

Comment:
```suggestion
    if (options.col == null || this.columns.length <= options.col) {
```

note that "== null" is the same as checking both null or undefined.  In general, we should prefer that over the `=== null|undefined`

Reply:
Why? I think the check for both `null` and `undefined` is much easier to read than the absence of one `=` is something you would easily miss during a review. Is this a new code style guideline that we're introducing and is it documented somewhere @ryantxu?

Reply:
I agree with Hugo. I strongly prefer the explicitness of Alexa's approach, which has less risks of someone who doesn't know the full history/context of Grafana coming in, misunderstanding the code, and reintroducing the bug.

Reply:
Fine by me --- we just need to be careful that treating null vs undefined differently is not going to work with anything coming from settings (that may be generated/processed by backend schemas and validation)